src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,46 +1,94 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+
+type Props = {
+  handleCloseModal: () => void;
+  loading: boolean;
+  setLoading: (con: boolean) => void;
+  setModalVisible: (con: boolean) => void;
+  selectedTodo: Todo;
+};
+
+export const TodoModal: React.FC<Props> = (
+  {
+    handleCloseModal,
+    loading,
+    setLoading,
+    setModalVisible,
+    selectedTodo,
+  },
+) => {
+  const [user, setUser] = useState<User | null>(null);
+
+  useEffect(() => {
```

Comment:
you don't need it

create here own `isLoading` state wich should be `true` only when we fetch user (before calling `getUser`)

Reply:
Not fixed,
please check again the previous mentor comments